frontend/src/components/common/header/index.css

```diff
@@ -150,3 +150,40 @@ a.dropdown-item {
     display: none;
   }
 }
+
+.resource-heading {
+  margin-top: 60px;
+  margin-bottom: 60px;
+  font-size: 48px;
```

Comment:
Font-size as pt- commented on an older branch

Reply:
`px` is fine here -> we want to use px over pt as pt are for print media.

Reply:
i think we can just use h1 here instead of adding a class.